src/main/java/org/pac4j/springframework/web/SecurityInterceptor.java

```diff
@@ -41,6 +42,8 @@
 
     private Config config;
 
+    private HttpActionAdapter httpActionAdapter = (code, webCtx) -> false;
```

Comment:
I don't think this will work. The `HttpActionAdapter` must apply the action on the web context. It should certainly be the `JEEHttpActionAdapter.INSTANCE`.